pkg/controller/eventing/subscription/reconcile_test.go

```diff
@@ -54,7 +54,7 @@ const (
 	testNS            = "testnamespace"
```

Comment:
should this, and references be changed too?

https://github.com/knative/eventing/blob/331bf4915aeace288f747758b7504272bacb578b/pkg/controller/eventing/subscription/reconcile_test.go#L51